frontend/src/layout/header/UserBox.tsx

```diff
@@ -0,0 +1,41 @@
+import { useTranslation } from "react-i18next";
+import { FiUser } from "react-icons/fi";
+import { useUser } from "../../User";
+
+import { ActionIcon } from "./ui";
+
+
+export const UserBox: React.FC = () => {
+    const { t } = useTranslation();
+    const user = useUser();
+
+    if (user === "unknown") {
+        return <Placeholder />;
+    } else if (user === "none") {
+        // TODO
+        return <Placeholder />;
+    } else {
+        // TODO
+        return (
+            <div css={{ height: "100%", display: "flex", alignItems: "center", gap: 16 }}>
+                <div>
+                    <div css={{ fontSize: 12, color: "var(--grey40)" }}>
+                        {t("user.logged-in-as")}
+                    </div>
+                    {user.displayName}
```

Comment:
This is suboptimal in regards to i18n. Think of RTL languages, for example. The user should be passed to `t` or `Trans` and interpolated by the translations.

Reply:
It's outdated. I removed the "logged in as". And the text would have been above the user name anyway. Otherwise good point.